packages/itwin/map-layers/package.json

```diff
@@ -96,15 +95,15 @@
   },
   "peerDependencies": {
     "@itwin/appui-abstract": "5.0.0-dev.111",
-    "@itwin/appui-react": "^4.17.6 || ^5.0.0",
-    "@itwin/components-react": "^4.17.6 || ^5.0.0",
-    "@itwin/core-bentley": "5.0.0-dev.111",
-    "@itwin/core-common": "5.0.0-dev.111",
-    "@itwin/core-frontend": "5.0.0-dev.111",
-    "@itwin/core-react": "^4.17.6 || ^5.0.0",
-    "@itwin/imodel-components-react": "^4.17.6 || ^5.0.0",
+    "@itwin/appui-react": "^5.0.0",
```

Comment:
same comment as https://github.com/iTwin/viewer-components-react/pull/1348#discussion_r2153105107 but also adding `@itwin/imodel-components-react` to that list.